doc/sphinx/changes.rst

```diff
@@ -14,6 +14,134 @@ Version 9.0
+Coq 8.20, providing compatitibility shims so that developments depending on Coq
+can be easily ported, see `Porting to The Rocq Prover`_  for details. The 9.0 version
+is based on a new single binary `rocq` that dispatches commands to previously
+separate binaries, a split and renaming of the standard library to `Stdlib` and
+an internal encoding of template-polymorphism in terms of the
+more general notion of sort polymorphism.
+
+We highlight some of the most impactful changes here:
+
+  - "The Rocq Prover" is the new official name of the project. We leave to users the
+  choice of renaming their projects to reflect this change, see `Renaming advice`_.
+
+  - A single `rocq` binary dispatches commands for compilation, read-eval-print-loop,
+  documentation building, dependency compatutation, etc... see :ref:`therocqcommands`.
+  It corresponds to the `rocq-runtime` `OCaml package <https://ocaml.org/p/rocq-runtime>`.
+  This is a bare-bones package not depending on any Gallina code.
+  CHECK LINK
+
+  - The `Coq` standard library has been split and renamed into two libraries. A
+  `Corelib` core library (an extended prelude corresponding to the `rocq-core` OCaml package,
+  binding to run `rocq-runtime`'s OCaml code).
+  and an `Stdlib` standard library (the `rocq-stdlib` opam package).
+  The `Stdlib` is now maintained `out of <https://github.com/rocq-prover/stdlib>`
+  the main `rocq` `repository <http://github/com/rocq-prover/rocq>`. FIX LINKS
+
+Notable breaking changes:
+
+  - TODO
+
+See the `Changes in 9.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 9.0 <https://rocq-prover.org/doc/v9.0/refman>`_, TODO fix links
+documentation of the 9.0 `core <https://rocq-prover.org/doc/v9.0/corelib>`_ and
+`standard <https://rocq-prover.org/doc/v9.0/stdlib>`_ libraries,
+`reference manual of the 9.0 standard library <https://rocq-prover.org/doc/v9.0/refman-stdlib>`_
+and `developer documentation of the 9.0 ML API <https://rocq-prover.org/doc/v9.0/api>`_
+are also available.
+
+The Rocq Prover comes with a new website the was developped by Matthieu Sozeau,
+Nicolas Tabareau and Théo Zimmermann, in collaboration with Bastien Sozeau of the
+`Noir Blanc Rouge <https://noirblancrouge.com/>` type foundry. The new website is
+a fork of the `OCaml.org <https://ocaml.org>` website developed by mainly by
+`Tarides <https://tarides.com/>`. The Rocq development team is thankful for
+their help and for open-sourcing their website. It includes full support for the
+`Rocq package archive <https://github.com/coq/opam>`, a responsive design and easy
+contributions through markdown files. The new identity, customized fonts and logo
+were designed by Bastien Sozeau, consulting for the Rocq development team. They are
+released under a permissive open-source license.
+
+Théo Zimmermann with help from Ali Caglayan and Jason Gross maintained
```

Comment:
not sure if Ali is still involved cc @Alizter

Reply:
Removed